<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentPage</class>
 <widget class="QWidget" name="PaymentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Store - Payment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
      background-color: #fef8ec;
      font-family: Arial;
    }
    QLabel {
      color: #333333;
    }
   </string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QLabel" name="storeLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Thanh toán </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="cartFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QVBoxLayout" name="cartLayout">
        <item>
         <widget class="QLabel" name="itemLabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Giỏ hàng (1 mặt hàng)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="itemTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
       QTableWidget {
         gridline-color: #e0e0e0;
       }
      </string>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Product Image</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Product Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Order Details</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QLabel" name="billDetailLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Chi tiết hóa đơn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="billFrame">
       <property name="styleSheet">
        <string>
         QFrame {
           border: 1px solid #e0e0e0;
           border-radius: 8px;
           padding: 20px;
         }
        </string>
       </property>
       <layout class="QVBoxLayout" name="billLayout">
        <item>
         <layout class="QFormLayout" name="billDetails">
          <item row="0" column="0">
           <widget class="QLabel" name="cartTotalLabel">
            <property name="text">
             <string>Tổng giỏ hàng</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="cartTotalValue">
            <property name="text">
             <string>1029.80</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="shippingLabel">
            <property name="text">
             <string>Vận chuyển</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="shippingValue">
            <property name="styleSheet">
             <string>color: #44cc44;</string>
            </property>
            <property name="text">
             <string>Free</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="totalLayout">
          <item>
           <widget class="QLabel" name="totalLabel">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Tổng</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalValue">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>1,683.80</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="placeOrderButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
              background-color: #a0c9c3;
              color: #333;
              border-radius: 4px;
              font-weight: bold;
              font-size: 14px;
            }
            QPushButton:hover {
              background-color: #657d81;
		 	  color: white;
            }
           </string>
          </property>
          <property name="text">
           <string>Thanh toán </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
              background-color: #e2b4b7;
              color: #333;
              border-radius: 4px;
              font-weight: bold;
              font-size: 14px;
            }
            QPushButton:hover {
              background-color: #bd637e;
		 	  color: white;
            }
           </string>
          </property>
          <property name="text">
           <string>Xuất hóa đơn</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
